README.aspnet.md

```diff
@@ -104,3 +114,4 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 View [license information](https://www.microsoft.com/net/dotnet_library_license.htm) for the software contained in this image.
 
 Windows Container images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
+

```

Comment:
Why add another blank line at the end of the readme?

Reply:
Not intentional, the powershell script used to update the readme uses Add-Content to write the content to the file, which is simply the existing content appended to the announcement, no new line added. I guess the Add-Content adds a new line. If this is a big deal I can change it. I used the script to update 194 files, hopefully I don't need to go edit this out of all of them.

Reply:
It turns out there is a -NoNewline switch for Add-Content, otherwise it does add a blank line for some weird reason. I'll rerun the script.